events/2022.03.Online/Architecture/Testimonials/siemens.html

```diff
@@ -0,0 +1,23 @@
+<div class="impl" id="impl-siemens">
+  <h3>Siemens AG</h3>
+  <div class="testimonial">
+    <p>
+      Siemens supports this specification as it allows manufacturers to attach
+      metadata to heterogeneous IoT devices and services in a uniform way. The
+      extensible model with a standardized serialization format in particular
+      remedies interoperability challenges with an installed base of long-lived
+      industrial devices. Furthermore, this specification is a central building
+      block for several digitalization topics such as device onboarding, device
+      management, digital twins, and data analytics.
+    </p>
+    <p>
+      Siemens commits to several open-source implementations of W3C Web of
+      Things, which are managed in the public Eclipse Thingweb and Eclipse
+      EdiTDor projects.
+    </p>
+    <p>
+      Siemens also implements WoT in its sayWoT! offering that can be used IoT
```

Comment:
```suggestion
      Siemens also implements WoT in its <a href="https://www.evosoft.com/digitalisierungsangebot/saywot/">sayWoT!</a> offering that can be used in IoT
```

Would it be possible to add this?